docs/pages/repo/docs/troubleshooting.mdx

```diff
@@ -75,6 +75,11 @@ directions even if you aren't planning to make a contribution.
 
 ## Common Pitfalls
 
+### Nested workspaces
+
+Turborepo does not support nested workspaces. Therefore, if you have a workspace that contains another workspace,
```

Comment:
Should we go into more detail as to what a nested workspace is?

Reply:
Or link to another page in handbook that explains what it is

Reply:
Do we have such a page? It might just be worth fleshing out here

Reply:
Not sure, just thinking that it's a general monorepo concept and as such can be defined somewhere else. We can do the easy thing and just flesh out here too, just a suggestion